domain/store/src/main/java/com/napzak/market/store/usecase/CheckNicknameDuplicationUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.napzak.market.store.usecase
+
+import com.napzak.market.store.repository.StoreRepository
+
+class CheckNicknameDuplicationUseCase(
+    private val storeRepository: StoreRepository,
+) {
```

Comment:
p1: 다음과 같이 수정하면 UseCase 모듈을 따로 구축할 필요가 없어집니다!

```suggestion
class CheckNicknameDuplicationUseCase @Inject constructor(
    private val storeRepository: StoreRepository,
) {
```

그리고 만약 `@Inject` 호출이 불가능해서 data에 UseCase 모듈을 만드신거라면 [이 PR](https://github.com/napzakmarket/Napzak-Android/pull/114)을 참고해주세요!